subprojects/plugins/src/integTest/resources/org/gradle/java/compile/IncrementalJavaCompileIntegrationTest/taskOutcomeIsUpToDateWhenNoRecompilationNecessary/build.gradle

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I would really prefer to have those files as some test fixture/helper methods in the test code instead of using `TestResources` to load them. My main reason for that is that I like to avoid "split logic", i.e. having the logic of the test (in this case the test setup) split over various places (Java code and resources).